docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -309,16 +312,65 @@
 
       when: helmv3_installed and helmv3_dm_exists.rc == 0
 
-    - name: Install Deployment Manager
-      shell: KUBECONFIG=/etc/kubernetes/admin.conf /usr/sbin/helm upgrade --install deployment-manager {% if helm_chart_overrides is defined %}--values {{ helm_chart_overrides }}{% endif %} {{ manager_chart }}
-      when: helmv3_installed
-
-    # Install or Upgrade DM-Monitor
-    - name: "Install or Upgrade DM-monitor"
-      shell: "ansible-playbook {{ dm_monitor_playbook }}"
-      become: yes
-      ignore_errors: yes
-      when: dm_monitor_playbook
```

Comment:
If we are planning for backward compatibility of this playbook we cannot remove this.
Otherwise, we have to make sure all relevant tasks of helmv2 and helmv3 upgrade / installation is going to be handled by system app framework and such tasks be deleted from the playbook (tasks such as deleting webhook service, restarting DM pod etc.).

Reply:
agree